File: Java/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag50/ExercicioA.java
package manzanoPag50;

import java.lang.Math;

/** a) Apresentar os quadrados dos números inteiros de 15 a 200 */

public class ExercicioA {

	public static void main(String[] args) {

		int contadora = 15;
		float resultado;

		do {
			resultado = (float) Math.pow(contadora, 2);
			System.out.println("O resultado do quadrado de " + contadora + " é " + resultado);
			contadora = contadora + 1;
		} while (contadora < 201);

	}

}

File: LogicaDeProgramacao(JavaScript)/exercicios pag 50(Manzano)/js/exercicioA.js
/** a) Apresentar os quadrados dos números inteiros de 15 a 200 */

var contadora = 15

do {
    var resultado = contadora**2
    alert("O resultado do quadrado de "+contadora+" é "+resultado)
    contadora = contadora + 1 
} while (contadora < 201)